### PR TITLE
Update redcap api module to docs v1 and v2

### PR DESCRIPTION
**Pull request by leemc-data-ed**
New metadata and macros added

### CODE REVIEW

File: using_redcap_api/using_redcap_api.md [hunk 1]
```diff
@@ -18,46 +27,35 @@ After completion of this module, learners will be able to:
...
 
-## Overview
-@comment
+@end
 
-**Is this module right for me?** @long_description
 
-**Estimated time to completion:** @estimated_time
+@version_history
+1.0.2: make it clear that you need to have R or Python installed
```
Comment (rosemm) [inline using_redcap_api/using_redcap_api.md:48]:
```suggestion
[1.0.2](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/a4ea7a7f1f9264dabe952b68941fc9f0f656c9fc/using_redcap_api/using_redcap_api.md#1): make it clear that you need to have R or Python installed
```

**Review by rosemm — CHANGES_REQUESTED** [review]
Just missing the link in version history, otherwise LGTM!

File: using_redcap_api/using_redcap_api.md (hunk 1 above)
Comment (leemc-data-ed) [reply using_redcap_api/using_redcap_api.md:48]: Oh yes, good catch!

**Comment by leemc-data-ed** [comment]
Okay, just added your suggestion @rosemm!

**Pull request merged**